### DISCUSSION TITLE
How to add the chartjs-plugin-zoom plugin like chartjs-plugin-datalabels plugin?

### DISCUSSION THREAD

**Discussion by mirzaeianfar** [kurkle/chartjs-chart-treemap#139] [Q&A] [answered]
I can not add chartjs-plugin-zoom to treemap chart
Please help me

**Comment by stockiNail** [comment]
I’m going to have a look in next days.

**Reply by mirzaeianfar** [reply]
I desperately need this feature
Thank you so much

**Reply by stockiNail** [reply]
After a first analysis, the issue is that the treemap controller is using always the chart area size to calculate the squares instead to use the scale size.

https://github.com/kurkle/chartjs-chart-treemap/blob/94efad8d9aea17668b96a28e9eae0bb4db932d7d/src/controller.js#L106

I'm doing some tests and sounds working. Let me create an issue

**Reply by stockiNail** [reply]
https://github.com/kurkle/chartjs-chart-treemap/issues/140

**Reply by stockiNail** [reply]
PR #141

**Comment by mirzaeianfar** [comment]
Perfect
Thank you so much

**Comment by stockiNail** [comment, marked as answer]
Feature has been released in version [2.1.0](https://github.com/kurkle/chartjs-chart-treemap/releases/tag/v2.1.0)
